app/views/newfromtemplate.html

```diff
@@ -16,6 +16,7 @@
                   {{ emptyMessage }}
                 </div>
                 <div class="osc-form" ng-show="template">
+                  <alerts alerts="alertsTop" hide-close-button="true"></alerts>
```

Comment:
I understand why you added this, but not sure we should have multiple alerts directives on this page. @jwforres Opinion? Were the alerts moved down for the quota changes?